bucket/python.json

```
@@ -1,35 +1,32 @@
 {
     "homepage": "https://www.python.org/",
-    "license": "https://docs.python.org/3/license.html",
+    "license": "Python-2.0",
     "version": "3.7.3",
+    "depends": "dark",
     "architecture": {
         "64bit": {
-            "url": "https://www.python.org/ftp/python/3.7.3/python-3.7.3-amd64.exe#/py3.exe",
+            "url": "https://www.python.org/ftp/python/3.7.3/python-3.7.3-amd64.exe",
             "hash": "md5:a2b79563476e9aa47f11899a53349383"
         },
         "32bit": {
-            "url": "https://www.python.org/ftp/python/3.7.3/python-3.7.3.exe#/py3.exe",
+            "url": "https://www.python.org/ftp/python/3.7.3/python-3.7.3.exe",
             "hash": "md5:ebf1644cdc1eeeebacc92afa949cfc01"
         }
     },
     "installer": {
-        "args": [
-            "/quiet",
-            "InstallAllUsers=$(@{$true=1;$false=0}[$global])",
-            "TargetDir=\"$dir\"",
-            "AssociateFiles=0",
-            "Shortcuts=0",
-            "InstallLauncherAllUsers=$(@{$true=1;$false=0}[$global])"
-        ],
-        "keep": true
+        "script": [
+            "&(file_path dark dark.exe) -nologo -x \"$dir\\_tmp\" \"$dir\\$fname\" | Out-Null",
+            "Remove-Item \"$dir\\_tmp\\AttachedContainer\\launcher.msi\" | Out-Null",
```

I would rather go with. ALso remove-Item do not show any data.

```suggestion
            "@('launcher.msi', 'path.msi', 'pip.msi') | ForEach-Object {",
            "    Remove-Item \"$dir\\_tmp\\AttachedContainer\\$_\"",
            "}",
```